lib/platform/azure/index.ts

```diff
@@ -572,29 +573,40 @@ export async function ensureComment({
 export async function ensureCommentRemoval({
   number: issueNo,
   topic,
+  content,
 }: EnsureCommentRemovalConfig): Promise<void> {
-  logger.debug(`ensureCommentRemoval(issueNo, topic)(${issueNo}, ${topic})`);
-  if (issueNo) {
-    const azureApiGit = await azureApi.gitApi();
-    const threads = await azureApiGit.getThreads(config.repoId, issueNo);
-    let threadIdFound = null;
+  logger.debug(
+    `Ensuring comment "${topic || content}" in #${issueNo} is removed`
+  );
+  if (!issueNo) {
+    return;
+  }
```

Comment:
```suggestion
```
@rarkins I think this should never happen?

Reply:
right

Reply:
ok, fixing it